File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/SavedTrack.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.os.Parcel;
import android.os.Parcelable;

/**
 * <a href="https://developer.spotify.com/web-api/object-model/#saved-track-object">Saved track object model</a>
 */
public class SavedTrack {
    public String added_at;
    public Track track;

    public SavedTrack() {
    }
}

File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/SavedTrack.kt
package com.timothy.spotifyarchitecture.retrofit.models

import android.os.Parcel
import android.os.Parcelable

/**
 * [Saved track object model](https://developer.spotify.com/web-api/object-model/#saved-track-object)
 */
class SavedTrack {
    var added_at: String = ""
    var track: Track = Track()
}
